src/mudlet.cpp

```diff
@@ -1985,56 +1910,56 @@ bool mudlet::openWindow(Host* pHost, const QString& name, bool loadLayout)
     }
 
     auto hostName(pHost->getName());
-    auto pC = pHost->mpConsole->mSubConsoleMap.value(name);
-    auto pD = pHost->mpConsole->mDockWidgetMap.value(name);
+    auto console = pHost->mpConsole->mSubConsoleMap.value(name);
```

Comment:
Can I make the observations that:
* the loss of the initial `p` is disguising (as far as I am concerned) that this and the next variable are pointers
* the noise from the variable renaming is hiding any changes in the functionality - hell, it isn't even obvious that anything *has* changed in this method...

Reply:
Valid points, but - a good Git client shows that only the variable was renamed in every line. Github's doesn't unfortunately, and if you confuse that the value is not a pointer, the compiler will tell you very fast otherwise, so no mistake can be made there.